website/.storybook/preview.js

```diff
@@ -12,6 +20,43 @@ export const parameters = {
   nextRouter: {
     Provider: RouterContext.Provider,
   },
+  msw: {
+    handlers: {
+      labels: [
+        rest.get("/api/valid_labels", (req, res, ctx) => {
+          return res(
+            ctx.json({
+              valid_labels: [
+                {
+                  name: "spam",
+                  display_text:
+                    "Seems to be intentionally low-quality or irrelevant",
+                  help_text: null,
+                },
+                {
+                  name: "fails_task",
+                  display_text:
+                    "Fails to follow the correct instruction / task",
+                  help_text: null,
+                },
+                {
+                  name: "not_appropriate",
+                  display_text: "Inappropriate for customer assistant",
+                  help_text: null,
+                },
+                {
+                  name: "violence",
+                  display_text:
+                    "Encourages or fails to discourage violence/abuse/terrorism/self-harm",
+                  help_text: null,
+                },
```

Comment:
could you move this object out of the function? the nesting will be reduced significantly, making it easier to read.

Reply:
Done